.github/workflows/merge-group.yml

```diff
@@ -6,6 +6,8 @@ on:
     types: [opened, synchronize, reopened, ready_for_review]
     branches: [master]
   merge_group:
+  # Manual trigger for early signal on local branches
+  workflow_dispatch:
```

Comment:
We should make this take a git reference (branch or commit name), especially if this job is going to generate bench comments on the commit itself.
```yaml
    inputs:
      label:
        description: "Git reference"
        required: true
        default: "master"

```

Reply:
Don't you have the commit already from the triggering branch that's required to start the workflow?

Reply:
The attributes above are *not* attributes of the job in general, they are, specifically, attributes of the `workflow_dispatch:` parameter. As in 
```yaml
  workflow_dispatch:
    inputs:
      label:
        description: "Git reference"
        required: true
        default: "master"
```
There is no "triggering" branch for a `workflow_dispatch`.

Reply:
What I mean is, the selected branch that the `workflow_dispatch` action runs on seems to have access to its respective `{{github.sha}}` and `{{github.ref}}` and not just that of `master`, as seen in https://github.com/samuelburnham/lurk-rs/actions/runs/6670651186/job/18130974696. Thus unless I'm missing something, adding the label to the action context is redundant